src/uu/tee/src/tee.rs

```diff
@@ -209,6 +194,6 @@ impl Read for NamedReader {
 }
 
 fn warn(message: &str) -> Error {
-    eprintln!("{}: {}", NAME, message);
-    Error::new(ErrorKind::Other, format!("{}: {}", NAME, message))
+    eprintln!("{}: {}", executable!(), message);
```

Comment:
I wonder if we could not use the show_warning function here instead?
https://github.com/uutils/coreutils/blob/d00356eab0ea98bab7ed5b0aa1e262673f3a9e3c/src/uucore/src/lib/macros.rs#L30

Reply:
done